vendor/assets/javascripts/slickgrid-editors/slickgrid-text.coffee

```diff
@@ -0,0 +1,43 @@
+###
+  Text Editor for Slickgrid
+  Because Slickgrid's text editor doesn't support option drop-down boxes.
+###
+
+@TextEditor = (args) ->
+  form = null
+  loadValue = null
+
+  if args.column.restrictions == ''
+    form = ($ '<input type="text" class="editor-text" />')
```

Comment:
The website is currently mid war on where the syntax should be 
``($ )`` or ``$()``

I'm imposing the second, becasuse I'm a :goat: